evap/staff/tests/test_views.py

```diff
@@ -2383,6 +2383,31 @@ def test_finishing_review_updates_results(self):
         self.assertEqual(len(results.questionnaire_results[0].question_results[1].answers), 1)
 
 
+class TestEvaluationTextAnswersSkip(WebTestStaffMode):
+    csrf_checks = False
+
+    def test_skip(self):
+        manager = make_manager()
+        student = baker.make(UserProfile, email="student@institution.example.com")
+        evaluation = baker.make(
+            Evaluation,
+            participants=[student],
+            voters=[student],
+            state="in_evaluation",
+            can_publish_text_results=True,
+        )
+        top_general_questionnaire = baker.make(Questionnaire, type=Questionnaire.Type.TOP)
+        baker.make(Question, questionnaire=top_general_questionnaire, type=Question.TEXT)
```

Comment:
Could you see if the test works without some of these models? (I would especially expect that these two are not necessary for the test to pass)

Reply:
When I remove the is_fully_reviewed check, this can be removed as well